Automattic/vip-go-mu-plugins
# Upgrading Jetpack 9.9 branch to release

## Description

We've upgraded Jetpack 9.9 branch from the beta version to its release one, running

```
./bin/update-jetpack-subtree.sh 9.9 9.9 
```

## Changelog Description

### Plugin Updated: Jetpack 9.9

Making Jetpack 9.9 release available.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
2. Set Jetpack version manually to 9.9
3. Test Jetpack functionality
